calico/getting-started/kubernetes/rancher.mdx

```diff
@@ -23,7 +23,7 @@ The geeky details of what you get:
 
 **Required**
 
-- A compatible [Rancher Kubernetes Engine cluster](https://rancher.com/docs/rke/latest/en/) with version 1.2.9 and later
+- A compatible [Rancher Kubernetes Engine cluster](https://rancher.com/docs/rke/latest/en/) with version 1.3
```

Comment:
Does this mean all 1.3.x releases? And not 1.4.x releases?

Reply:
Yes, it means just 1.3.x release. We have to revalidate all "next" releases with RKE.